packages/bcsc-core/package.json

```diff
@@ -80,7 +80,7 @@
     "prettier": "^3.0.3",
     "react": "19.0.0",
     "react-native": "0.78.2",
-    "react-native-builder-bob": "^0.40.6",
```

Comment:
is there a reason to downgrade react-native-builder-bob from 0.40.6 to ^0.39.1?

Reply:
I was having issues building this package with the 0.40 version.

- react-native-builder-bob version 0.40.x introduced a dependency on arktype ^2.1.15
- arktype version 2.x is an ES Module ("type": "module" in package.json)
- react-native-builder-bob was trying to import arktype using CommonJS require() statements
- This created an incompatibility between ESM and CommonJS module systems

Happy to use another solution if there is one